SpatialGDK/Source/SpatialGDK/Public/SpatialView/ViewCoordinator.h

```diff
@@ -16,17 +17,33 @@ class ViewCoordinator
 
 	~ViewCoordinator();
 
-	// Moveable, not copyable.
+	// Non-copyable.
 	ViewCoordinator(const ViewCoordinator&) = delete;
-	ViewCoordinator(ViewCoordinator&&) = delete;
+	ViewCoordinator(ViewCoordinator&&) = default;
 	ViewCoordinator& operator=(const ViewCoordinator&) = delete;
-	ViewCoordinator& operator=(ViewCoordinator&&) = delete;
+	ViewCoordinator& operator=(ViewCoordinator&&) = default;
 
 	void Advance();
-	const ViewDelta& GetViewDelta();
-	const EntityView& GetView();
+	const ViewDelta& GetViewDelta() const;
+	const EntityView& GetView() const;
 	void FlushMessagesToSend();
 
+	// Create a subview with the specified tag, filter, and refresh callbacks.
+	FSubView& CreateSubView(const Worker_ComponentId& Tag, const FFilterPredicate& Filter,
+							const TArray<FDispatcherRefreshCallback>& DispatcherRefreshCallbacks);
+	// Create a subview with no filter. This also means no refresh callbacks, as no change from spatial could cause
+	// the subview's filter to change its truth value.
+	FSubView& CreateUnfilteredSubView(const Worker_ComponentId& Tag);
+	// Force a refresh of the given entity ID across all subviews. Used when local state changes which could
+	// change any subview's filter's truth value for the given entity. Conceptually this can be thought of
+	// as marking the entity dirty for all subviews, although the refresh is immediate.
+	// Note: It would be possible to only refresh the subviews that require refreshing instead of globally refreshing.
```

Comment:
It *is* possible to refresh just the subviews you want to though with `FSubView::RefreshEntity`

Reply:
Yup, that's what I meant by having systems understand which subviews need refreshing. Can rejig this to be clearer if that's not what you understood